packages/project-utils/bundling/function/buildFunction.js

```diff
@@ -47,7 +50,9 @@ module.exports = options => {
                 if (messages.errors.length > 1) {
                     messages.errors.length = 1;
                 }
-                return reject(new Error(messages.errors.join("\n\n")));
+
+                console.error(messages.errors.join("\n\n"));
```

Comment:
Is this console log necessary here?

Reply:
I replaced `reject` with error with a simple console log and `reject` with no param, because it gives cleaner output. Otherwise in case of error you would see a stack trace leading you to this exact line, which is not very useful for anything.

But yes, you are right, I can do a reject with string just the same way as in code above.